challenge_10/python/shaowen/solution.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 11 09:21:20 2017
+
+@author: Shaowen Liu
+"""
+def solution(strr):
+    """check if all brackets closed properly"""
+    
+    closerof = {'{':'}', '[':']','(':')'}
+    stack = []
+    
+    for i in list(strr):
```

Comment:
tks!
In my mind, string is not an efficient date structure, so I used to convert it to list......
Well, in this case, absolutely not necessary!